content/sensu-go/6.2/operations/deploy-sensu/use-federation.md

```diff
@@ -312,24 +289,24 @@ spec:
   }
 }
 {{< /code >}}
-
 {{< /language-toggle >}}
 
-To configure this etcd replicator on your `gateway` cluster, use `sensuctl config view` to verify that `sensuctl` is configured to talk to a `gateway` cluster API. Reconfigure `sensuctl` if needed.
+2. Run `sensuctl config view` and verify that `sensuctl` is configured to talk to a `gateway` cluster API.
+Reconfigure `sensuctl` if needed.
 
-Write these EtcdReplicator definitions written to disk and use `sensuctl create -f` to apply them to the `gateway` cluster.
+3. Write the `AlphaClusterRoleBindings` and `BetaClusterRoleBindings` EtcdReplicator definitions to disk.
 
-For a consistent experience, repeat the `ClusterRoleBinding` example in this guide for `Role`, `RoleBinding` and `ClusterRole` resource types.
-The [etcd replicators reference][2] includes [examples][9] you can follow for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources.
+4. Use `sensuctl create -f` to apply the `AlphaClusterRoleBindings` and `BetaClusterRoleBindings` EtcdReplicator definitions to the `gateway` cluster.
 
-To verify that the EtcdReplicator resource is working as expected, reconfigure `sensuctl` to communicate with the `alpha` and then `beta` clusters and run the following command for each:
+5. For a consistent experience, create an EtcdReplicator resource for `Role`, `RoleBinding` and `ClusterRole` resource types.
+The [etcd replicators reference][2] includes [examples][9] you can follow for `Role`, `RoleBinding`, and `ClusterRole` resources.
```

Comment:
Anthony called out this instruction yesterday and thought it might have referred to repeating the previous section (currently https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/use-federation/#step-3-add-a-cluster-role-binding-and-user) for role, role binding, and cluster role resources. After reading the instructions myself, I think it refers to creating etcd replicators for the role, role binding, and cluster role resources. But I'm hoping the review can include testing this to see if it makes sense.

Reply:
I created these additional etcd replicators on our skunkworks instance but it didn't make a difference - I cannot see either of the clusters' namespaces when logged into the federated dashboard as the `federation-viewer` read-only user - although I can with the default `admin` user. 
Whether this means the instructions are incorrect or there is some other issue with the skunkworks issue is hard for me to say.

Reply:
@agoddard just wondering if you happened to try the federation example in skunkworks and can confirm the issue Mike and I are running into here.